xchange-deribit/src/main/java/org/knowm/xchange/deribit/v2/dto/DeribitResponse.java

```diff
@@ -0,0 +1,86 @@
+package org.knowm.xchange.deribit.v2.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** V represents result class of the queried endpoint */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DeribitResponse<V> {
+
+  @JsonProperty("jsonrpc")
+  private String jsonRPC;
+
+  @JsonProperty("id")
+  private long id;
+
+  @JsonProperty("result")
+  private V result;
+
+  @JsonProperty("error")
+  private DeribitError error;
+
+  @JsonProperty("testnet")
+  private boolean testnet;
+
+  @JsonProperty("usIn")
+  private long usIn;
+
+  @JsonProperty("usOut")
+  private long usOut;
+
+  @JsonProperty("usDiff")
+  private long usDiff;
+
+  public DeribitResponse() {}
+
+  public DeribitResponse(
+      String jsonRPC,
+      long id,
+      V result,
+      DeribitError error,
+      boolean testnet,
+      long usIn,
+      long usOut,
+      long usDiff) {
+    this.jsonRPC = jsonRPC;
+    this.id = id;
+    this.result = result;
+    this.error = error;
```

Comment:
if there is an error then throw a ExceptionalReturnContentException here instead of saving the error DTO here

this will force the client to parse and throw a DeribitException  of which I mentioned in the previous comment